src/template.js

```diff
@@ -15,7 +15,9 @@ const thumbnailFileExtension = ".png"
     return imageKey;
   }
   
-  return templateName?.toLowerCase().trim().replace(/\/|\s+/g, "-") || "no-key"
+  return !templateName ? 
+    "no-key" : 
+    templateName.toLowerCase().trim().replace(/\/|\s+/g, "-")
```

Comment:
Not sure what this changes - the optional chaining in the existing code already handled the case where template name doesn't exist. Was there a case where the original code didn't work?

Reply:
The mystery behind why publishing didn't work! Haha the node script that must not be set up to use new features. Makes sense to change this though rather than look into updating the script.